app/cosmos/index.js

```diff
@@ -1,9 +1,12 @@
 const { preferencesDatabase } = require('./database')
 
-const cosmos = async () => {
-  const cosmos = {}
-  cosmos.preferencesDatabase = await preferencesDatabase()
-  return cosmos
+const cosmosInstance = async () => {
+  try {
+    const cosmos = {}
+    cosmos.messagesDatabase = await preferencesDatabase()
```

Comment:
Looks like a copy paste error here?  Database name is referencing messages instead of preferences. This makes it fail when sending a message to the service.

Did you test locally by sending a service bus message to the topic? Can use the service bus explorer on the azure portal or there is docker image John created that you can use:
https://hub.docker.com/r/johnwatson484/azure-service-bus-test-client

Reply:
Yes, this is my oversight! Thanks for pointing this out :) I have tried to use this tester, but failed to find all credentials required to perform this test